.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,17 @@
+**Reviewer Checklist**
+<!-- Check [Expectations from a PR](/CONTRIBUTING.md#expectations-from-a-pr) for the details -->
+
+- [ ] PR Message
```

Comment:
worth to add also the commit message itself, that should be clear, concise and comprehensive.

Reply:
Added.